integration/matchers/flux.go

```diff
@@ -335,7 +335,7 @@ func assertValidHelmOperatorDeployment(fileName string) {
 			Expect(deployment.Spec.Template.Spec.Containers).To(HaveLen(1))
 			container := deployment.Spec.Template.Spec.Containers[0]
 			Expect(container.Name).To(Equal("helm-operator"))
-			Expect(container.Image).To(Equal("docker.io/fluxcd/helm-operator:1.0.0"))
+			Expect(container.Image).To(ContainSubstring("docker.io/fluxcd/helm-operator"))
```

Comment:
There's also `HavePrefix`:
```suggestion
			Expect(container.Image).To(HavePrefix("docker.io/fluxcd/helm-operator"))
```

Reply:
Oh nice one, thanks!